Warhammer-Fantasy-Roleplay-2nd-Edition/translations/fr.json

```diff
@@ -137,43 +161,64 @@
   "magic":"Magie",
   "insanity-points":"Points de Folies",
   "fate-points":"Points de Destin",
+
   "career-skills":"Compétences de Carrière",
   "career-talents":"Talents de Carrière",
   "career-exits":"Débouchés",
+
   "second-career":"Seconde Carrière",
   "third-career":"Troisième Carrière",
   "fourth-career":"Quatrième Carrière",
   "fifth-career":"Cinquième Carrière",
+
   "melee-weapons":"Armes de Mélée",
+
   "abbreviate-proficiency":"Maît?",
+
   "weapon":"Arme",
   "craftsmanship":"Qualité",
+  "poor": "Médiocre",
+  "common": "Commune",
+  "good": "Bonne",
+  "best": "Exceptionnelle",
   "ws-bonus":"Bonus CC",
   "to-hit":"Dés Min",
   "damage":"Dégats",
-  "impact":"Percutante",
-  "defensive":"Defensive",
+  "impact":"Perc",
+  "defensive":"Def",
   "qualities":"Attributs",
+
   "attack":"Attaque",
   "parry":"Parade",
+
   "abbreviate-attack":"Atk",
   "abbreviate-damage":"Dmg",
   "abbreviate-parry":"Prd",
+
   "ranged-weapons":"Arme à Distance",
   "bs-bonus":"Bonus de CT",
   "type":"Type",
+  "shoot": "Tir",
+  "throw": "Jet",
   "reload":"Rechargement",
+  "free": "Libre",
+  "half": "Demis",
+  "full": "Complète",
+  "full-number": "{{0}} Complète(s)",
```

Comment:
@pjbehr87  This is difficult because the plural is not handled the same way between languages.
For example, the 0 is plural in English but not in French 
en: 0 cars, 1 car, 2 cars
fr: 0 voiture, 1 voiture, 2 voitures